### PR TITLE
res.locals edit

### PR DESCRIPTION
**Pull request by pllearns**
- I wasn't sure if this is entirely necessary for most, but I wanted to add a note explicitly noting that res.locals takes precedent over app.locals. 
- This might be too much detail, but I thought it might help understanding the use of both app.locals and res.locals.

### CODE REVIEW

File: _includes/api/en/4x/res-locals.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ this property is identical to [app.locals](#app.locals).
 This property is useful for exposing request-level information such as the request path name,
 authenticated user, user settings, and so on.
 
+Note: a `res.locals` object takes immediate precedent over `app.locals` during the request/response cycle.
```
Comment (crandmck) [inline _includes/api/en/4x/res-locals.md:10]: I would prefer "... takes precedence over ... "

**Comment by crandmck** [comment]
This looks OK to me (except for minor wording quibble). 
@dougwilson PTAL.

**Comment by dougwilson** [comment]
I don't really understand what the note is trying to add to the `res.locals` documentation; it may just be the way it's worded is throwing me off here.

**Comment by crandmck** [comment]
@pllearns Sorry I forgot to submit my review comment.... PTAL.

**Comment by dougwilson** [comment]
PR seems stale; no response in a long while to the review comment.

**Pull request closed without merging**